assets/app/styles/_list.less

```diff
@@ -0,0 +1,22 @@
+.confirm-delete {
+  background: @state-warning-bg;
+  color: @state-warning-text;
+  margin-top: 0px;
+  padding: 10px 25px;
+  position: relative;
+}
+
+.delete-button {
```

Comment:
@jwforres which one? the delete-button (trash icon) or confirm-delete (the buttons that say cancel and delete)?